index.html

```diff
@@ -14,29 +14,386 @@
+                </tr>
+              </tbody>
+
+            </table>
+          </div>
         </div>
-        <div class="section" id="project">
-          Learn more about
-          <a id="lokole" href="http://donate.ascoderu.ca/">Lokole</a>
-          our project for the Congo.
+
+      </div>
+      <div class="transparent-section-footer">
+        <a href="#help-us" class="btn btn-lg btn-next" data-toggle="tooltip" title="next page" data-placement="left">
+          <span class="glyphicon glyphicon-menu-down"></span>
+        </a>
+      </div>
+    </div>
+
+    <div class="section" data-anchor="help-us">
+      <div class="container-fluid">
+        <p class="extra-large">Do you want to help us?</p>
+        <div class="row-wrap-layout">
+          <a class="no-style" href="https://github.com/ascoderu" target='_blank'>
+            <div class="action-thumbnail" data-toggle="tooltip" title="go to github" data-placement="top">
+              <i class="glyphicon glyphicon-console"></i>
+              <p>&lt;code/&gt;</p>
+            </div>
+          </a>
+
+          <a class="no-style" href="https://www.transifex.com/ascoderu/opwen-webapp/" target='_blank'>
+            <div class="action-thumbnail" data-toggle="tooltip" title="go to transifex account" data-placement="top">
+              <i class="glyphicon glyphicon-globe"></i>
+              <p>translate</p>
+            </div>
+          </a>
+
+          <a class="no-style" href="mailto:info@ascoderu.ca">
+            <div class="action-thumbnail" data-toggle="tooltip" title="send us an email" data-placement="top">
+              <i class="glyphicon glyphicon-question-sign"></i>
+              <p>ask</p>
+            </div>
+          </a>
+
+          <a class="no-style" href="https://donate.ascoderu.ca/" target='_blank'>
+          <div class="action-thumbnail" data-toggle="tooltip" title="go to our donate page" data-placement="top">
+            <i class="glyphicon glyphicon-usd"></i>
+            <p>donate</p>
+          </div>
+        </a>
+        </div>
+        <a class="no-style medium" href="https://youtu.be/SKtfNZdK_Vw" target='_blank' id="trans-tutorial">
+          <em>see translation tutorial</em>
+        </a>
+
+      </div>
+      <div class="transparent-section-footer">
+        <a href="#news" class="btn btn-lg btn-next" data-toggle="tooltip" title="next page" data-placement="left">
+          <span class="glyphicon glyphicon-menu-down"></span>
+        </a>
+
+      </div>
+    </div>
+
+    <div class="section" data-anchor="news">
+      <div class="container-fluid">
+        <p class="large">What do people say about us?</p>
+        <a class='no-style' href='https://www.howwemadeitinafrica.com/congolese-inventor-solution-get-rural-africa-emailing/59256/' target='_blank'>
+          <div class='box'>
+            <em>Congolese inventor has a solution to get rural Africa emailing</em>,
+            <span>25th Jul 2017 - howwemadeitinafrica.com</span>
```

Comment:
Typography nitpick: technically a em-dash `&mdash;` would be more appropriate here than a hyphen. [More information](http://www.thepunctuationguide.com/em-dash.html).